website/docs/registries/packaging.md

```diff
@@ -0,0 +1,33 @@
+# Packaging your project
+
+When you want to share your package over package registry, it needs to be turned into an archive first.
```

Comment:
This is a good in-depth description although i think a basic outline first "what" and after this "why"/"how" would make it more clear

Reply:
Discussed offline, addressed in 854ec0a6f796883eca8f18a1b5db85a854367f43